pip_update.sh

```diff
@@ -1,9 +1,9 @@
 #!/bin/bash
 pip3 uninstall featureform -y
 rm -r client/dist/*
-cd dashboard && npm run build
-cd ../
-mkdir -p client/src/featureform/dashboard/
-cp -r dashboard/out client/src/featureform/dashboard/
+# cd dashboard && npm run build
```

Comment:
Should these be commented?

Reply:
mistake, commented to make rebuild faster since dashboard wasn't necessary to test. I'll make a seperate script